apis/config/v1beta1/groupversion_info.go

```diff
@@ -31,6 +31,10 @@ var (
 	// SchemeBuilder is used to add go types to the GroupVersionKind scheme
 	SchemeBuilder = &scheme.Builder{GroupVersion: GroupVersion}
 
+	// localSchemeBuilder is used to register autogenerated conversion and defaults functions
+	// It is required by ./zz_generated.conversion.go and ./zz_generated.defaults.go
+	localSchemeBuilder = &SchemeBuilder.SchemeBuilder
```

Comment:
What is the reason to declare `localSchemeBuilder`, maybe just refer via `SchemeBuilder.SchemeBuilder`  (as before)?

Reply:
It seems the convention is to declare that `localSchemeBuilder` variable. This is what's done in /apis/visibility/v1alpha1` and also in Kubernetes components, like the scheduler for example. So it's mostly for consistency.

Reply:
sgtm